packages/insomnia-importers/package.json

```diff
@@ -8,21 +8,36 @@
   "bugs": {
     "url": "https://github.com/kong/insomnia"
   },
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
   "bin": {
-    "insomnia-import": "./bin/cli"
+    "insomnia-import": "./dist/src/cli"
   },
   "scripts": {
+    "bootstrap": "npm run build",
+    "lint": "eslint . --ext .js,.ts,.tsx --ignore-path='../../.eslintignore'",
+    "clean": "tsc --build tsconfig.build.json --clean",
+    "postclean": "rimraf dist",
+    "build": "tsc --build tsconfig.build.json",
     "test": "jest --silent"
   },
   "dependencies": {
-    "apiconnect-wsdl": "^1.8.29",
+    "apiconnect-wsdl": "^1.8.31",
     "change-case": "^4.1.1",
-    "commander": "^2.20.0",
+    "commander": "^7.1.0",
     "lodash": "^4.17.15",
-    "shell-quote": "^1.6.1",
-    "swagger-parser": "^6.0.5",
-    "yaml": "^1.5.0"
+    "shell-quote": "^1.7.2",
+    "@apidevtools/swagger-parser": "10.0.2",
```

Comment:
Another big version jump from 6.x to 10.x

Reply:
please consult the scratchpad where the reason for this change was already described https://docs.google.com/document/d/1nDYsQnWMHRNNIzhwMjHjfxGJf3lAO_zuumGwhi89jIo/edit#